src/pipelines/epidemiology/config.yaml

```diff
@@ -93,13 +93,15 @@ sources:
       job_group: all
       deferred: true
 
-  # Data sources for AT level 2
-  - name: pipelines.epidemiology.xx_covid19_eu_data.Covid19EuDataSource
+  - name: pipelines.epidemiology.at_2_authority.AustriaLevel2DataSource
```

Comment:
I would suggest renaming the file to just `at_authority`, and if we have multiple authoritative sources we can add them to the same file (like we do for the UK)

```suggestion
# Data sources for AT levels 1 + 2
  - name: pipelines.epidemiology.at_authority.AustriaDataSource
```

Reply:
Sounds good, will do (there's also the L3 data, but I could process that in the same data source, possibly in another commit)

Reply:
Right, I forgot about the L3 data. Yes, add it to the same file whenever that's ready — it can be in a different PR.

Reply:
In that case you can keep the name of the class, or even something like `AustriaProvincesDataSource` (or whatever they are called)

Reply:
If I'm going to be adding L3 data later in the same class (is that what you're suggesting, rather than having it in a different class like I was initially planning on?), I could just call it AustriaDataSource.

Reply:
Sorry if I wasn't clear. We normally add a different class to the same file. See for example: https://github.com/GoogleCloudPlatform/covid-19-open-data/blob/main/src/pipelines/epidemiology/gb_authority.py

Reply:
If it's very easy to do it in the same class, then of course there's no need to duplicate the work and add a second class!

Reply:
Renamed to at_authority.